hack/check-EOF.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. && pwd)"
+
+CMD="/usr/bin/find $ROOTDIR/"
+
+EXCLUDE_DIR=(
+    vendor
+    output
+    _tools
+    .git
+    static/tidb-operator-overview.png
+)
+
+for DIR in ${EXCLUDE_DIR[@]}; do
+    CMD="$CMD -path $ROOTDIR/$DIR -prune -o"
+done
+
+FILELIST=$($CMD -type f -print)
```

Comment:
I suggest you use `\` to concatenate multiple lines into one command, easier to read, e.g.

```
find . -type f -not \( -path './vendor/*' \
    -o -path './output/*' \
    -o -path './_tools/*' \
    -o -path './.git/*' \
    -o -path './*/.terraform/*' \
    -o -path '*.png' \
    \)
```

- `-type f` limit files only
- `\(` and `\)` to group expressions, etc.